skeletor/export/js.config.js

```diff
@@ -10,7 +10,7 @@ module.exports = {
 				input,
```

Comment:
[fix] Can you also add in a rollup config for legacy bundles in this export configuration?

Reply:
Done!